code.js

```diff
@@ -3,7 +3,7 @@ function isProduction(url) {
 }
 
 function isLocal(url) {
-    return url && url.indexOf('//localhost') > 0;
+    return url && url.match(/(\/\/localhost|github\.io)/);
```

Comment:
If it's github.io then the following ternary in `getBaseUrl()` should return the else section. However, by this change, it won't handle URLs like this `https://michaelmueller-binary.github.io/websockets/api/`. Right?

https://github.com/binary-com/websockets/blob/4819c31efa50887b473af218bc31b1778831b5c7/code.js#L11

Reply:
Probably my fault for the way I set up git hub pages  my urls looks like `https://michaelmueller-binary.github.io/api/`